enterprise-ws-spec/src/main/asciidoc/Overview.adoc

```diff
@@ -175,22 +175,25 @@ JAX-WS 2.2 adds a complete Web Services addressing support as specified
 in Web Services Addressing 1.0 - Core, Web Services Addressing 1.0 -
 Soap Binding, and Web Services Addressing 1.0 - Metadata.
 
+Jakarta XML Web Services 3.0 renames the JAX-WS API packages from `javax.*`
+to `jakarta.*`. Beginning in Java SE 11, the JAX-WS APIs are no longer
+available as part of the Java SE class library.
+
 === Interoperability
 
-This specification extends the interoperability requirements of the Java
+This specification extends the interoperability requirements of the Jakarta
 EE platform by defining interoperability requirements for products that
-implement this specification on top of Java EE™. The interoperability
+implement this specification on top of Jakarta EE. The interoperability
 requirements rely on the interoperability of existing standards that
 this specification depends on.
 
 The specification builds on the evolving work of the following JSRs and
 specifications:
 
-* Java™ API for XML-based RPC (JAX-RPC) 1.1
-* Java™ API for XML-based Web Services (JAX-WS) 2.2
-* Java Enterprise Edition Specification
-* Enterprise JavaBeans Specification
-* Java Servlet Specification
+* Jakarta API for XML-based Web Services
```

Comment:
I would not remove JAX-RPC in this case and 'Jakarta API for XML-based Web Services' is wrong. Here should be:
```
* Jakarta XML-RPC
* Jakarta XML Web Services
```

Reply:
I do not care much about 'Specification' suffix is used or not